src/components/card/Card.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Card } from './Card';
+import { Button } from '../Button/Button';
+import image1 from '../../assets/image1.svg';
```

Comment:
you can add and index in /assets/ 

export all of the assests from there

Reply:
Ok, for the moment, this assets aren`t necessary